packages/docs/components/ColorCards/ColorCard.tsx

```diff
@@ -0,0 +1,83 @@
+import { Flex, Link, Small, Text } from '@bigcommerce/big-design';
+import React, { useContext } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+
+import { ColorDetails } from './availableColors';
+
+const StyledColor = styled(Flex)`
+  height: ${({ theme }) => theme.helpers.remCalc(160)};
+  width: ${({ theme }) => theme.helpers.remCalc(184)};
+  border-radius: 4px;
+`;
+
+interface ColorCardProps {
+  colorCard: ColorDetails;
+}
+
+const ColorCard: React.FC<ColorCardProps> = ({ colorCard }) => {
+  const { colors } = useContext(ThemeContext);
+  const hasDescription = colorCard.description;
+  const hasHexagesimalColor = colors[colorCard.name].startsWith('#');
+  const hasSecondaryTextColor = colorCard.secondaryTextColor !== undefined;
+
+  const renderContrast = () => {
+    if (hasSecondaryTextColor) {
+      return (
+        <Flex alignItems="flex-end">
+          <Text marginRight="xSmall" marginBottom="none" color={colorCard.textColor}>
+            {colorCard.contrast}
+          </Text>
+          <Text marginBottom="none" color={colorCard.secondaryTextColor}>
+            {colorCard.contrast}
+          </Text>
+        </Flex>
+      );
+    }
+
+    return (
+      <Text margin="none" color={colorCard.textColor}>
+        {colorCard.contrast}
+      </Text>
+    );
+  };
+
+  return (
+    <StyledColor
+      backgroundColor={colorCard.name}
+      justifyContent={hasHexagesimalColor ? 'space-between' : 'baseline'}
+      flexDirection={'column'}
+      border="box"
+    >
+      <Text margin="small" color={colorCard.textColor} bold>
+        {getColorLabel(colorCard.name)}
+      </Text>
+
+      {hasDescription && (
+        <Small color={colorCard.textColor} margin={'small'}>
+          {colorCard.description?.text}{' '}
+          {colorCard.description?.href && (
+            <Link href={colorCard?.description.href} target="_blank">
+              <Small color="inherit">Read more</Small>
+            </Link>
+          )}
+        </Small>
+      )}
+
+      {hasHexagesimalColor && (
+        <Flex margin="small" justifyContent={'space-between'} flexWrap={'wrap'}>
```

Comment:
Same above, use `"` for string props.

Note to self: This should be a lint rule.